Makefile

```
@@ -66,19 +66,17 @@ INTEG_TEST_NDC_ROOT    := ./host/ndc
 PROTO_ROOT := proto
 PROTO_FILES = $(shell find ./$(PROTO_ROOT)/internal -name "*.proto")
 PROTO_DIRS = $(sort $(dir $(PROTO_FILES)))
-PROTO_IMPORTS := -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(GOPATH)/src/github.com/temporalio/gogo-protobuf/protobuf
+PROTO_IMPORTS = -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(shell cd build && go list -m -f '{{.Dir}}' github.com/temporalio/gogo-protobuf)/protobuf
```

Just curious, does this work?
```suggestion
PROTO_IMPORTS = -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(shell cd build && go list -f '{{.Dir}}' github.com/temporalio/gogo-protobuf/protobuf)
```

---

```
$ go list -f '{{.Dir}}' github.com/temporalio/gogo-protobuf/protobuf
no required module provides package github.com/temporalio/gogo-protobuf/protobuf; to add it:
        go get github.com/temporalio/gogo-protobuf/protobuf
$ go get github.com/temporalio/gogo-protobuf/protobuf
go: module github.com/temporalio/gogo-protobuf@upgrade found (v1.22.1), but does not contain package github.com/temporalio/gogo-protobuf/protobuf
```
:-(